cmd/hz/app/app.go

```diff
@@ -318,9 +318,9 @@ func GenerateLayout(args *config.Argument) error {
 		}
 	} else {
 		// generate by customized layout
-		sp := strings.Split(args.CustomizeLayout, ":")
+		sp := strings.Split(args.CustomizeLayout, "|")
```

Comment:
'customize package' may also have this problem

Reply:
`customize package` use `GetVerticalBarPair` to split